.github/workflows/build-latest-preview.yml

```
@@ -23,7 +23,7 @@ jobs:
 
   upload-preview:
     needs: build
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
```

Let's keep this versioned, and use `ubuntu-24.04` which currently it's the same as `ubuntu-latest`. Setting the version explicitly will prevent accidental breakage when `ubuntu-latest` points to a different version.

```suggestion
    runs-on: ubuntu-24.04
```

---

Done :+1: